src/Compilers/CSharp/Portable/Symbols/SubstitutedMethodSymbol.cs

```diff
@@ -219,14 +219,6 @@ public sealed override Symbol AssociatedSymbol
             }
         }
 
-        public sealed override bool ReturnsVoid
```

Comment:
>public sealed override bool ReturnsVoid [](start = 8, length = 39)

Is this covered by WrappedMethodSymbol?

Reply:
Yes.

---
In reply to: [382294420](https://github.com/dotnet/roslyn/pull/41728#discussion_r382294420) [](ancestors = 382294420)